### ISSUE TITLE
System.ComponentModel.Win32Exception: An error occurred trying to start process 'C:\Program Files\VidCoder-Beta\unins000.exe' with working directory 'C:\Users\june\AppData\Local\VidCoder.Beta\app-7.8.0'. The system cannot find the file specified.

### ISSUE THREAD

**Issue by Janie69** [RandomEngy/VidCoder#977]
### What were you doing when VidCoder crashed?

Installation

### Crash details

```text
I have run VidCoder beta for several years without problems. Recently when I closed the application, a message stated that an update had been downloaded, would I like to install it? I selected Yes but the installation failed. I then uninstalled the application, downloaded the current installer and ran it, resulting in the following error message:
"squBBD2.tmp.exe - System Error
The program can't start because api-ms-win-core-winrt-l1-1-0.dll is missing from your computer. Try reinstalling the program to fix this problem."
I then downloaded a copy of the missing dll into the C:\Windows\SysWOW64 directory and ran the VidCoder beta installer again, resulting in the following error message:
"Setup Error
An error occurred while running Setup: Process exited with error code: 3221225781. Please contact the application author."

OS: Windows 7 Professional
```

**Comment by RandomEngy** [comment]
VidCoder is not supported on Windows 7. But someone else was having a similar problem on Windows 7: https://github.com/RandomEngy/VidCoder/discussions/974

**Comment by Janie69** [comment]
Thanks. I guess when you're running an older OS, then there is a risk that any software update that you download might turn out to be incompatible with your OS. With any luck, I might be able to track down a copy of Vidcoder 6.45 and install that.

**Comment by RandomEngy** [comment]
Actually, this was a different issue and is fixed with v7.11.
